maven-java/kalix-java-protobuf-parent/pom.xml

```diff
@@ -348,6 +348,7 @@
                         <!-- run *IntegrationTest with failsafe -->
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.2</version>
```

Comment:
The lack of a version was causing classpath issues. 

This is the version we used initially, but latest version is now 3.2.5.